emission/tests/modellingTests/TestSupportVectorMachine.py

```diff
@@ -0,0 +1,460 @@
+import unittest
+import emission.analysis.modelling.trip_model.support_vector_machine as eamts
+import emission.tests.modellingTests.modellingTestAssets as etmm
+import logging
+import pandas as pd
+import random
+
+
+class TestSupportVectorMachine(unittest.TestCase):
+
+    def setUp(self) -> None:
+        logging.basicConfig(format='%(asctime)s:%(levelname)s:%(message)s',
+        level=logging.DEBUG)
+
```

Comment:
Aren't these very similar to the `TestGradientBoostedDecisionTree.py`? Can't you just refactor into one set of tests where you pass in the model? Except for the partial fit (which makes sense as a separate test), it seems like both of them have the same inputs and outputs.